mirgecom/flux.py

```diff
@@ -35,6 +44,40 @@
 import numpy as np  # noqa
 
 
+def num_flux_lfr(f_minus, f_plus, q_minus, q_plus, lam, **kwargs):
+    """Lax-Friedrichs/Rusanov low-level numerical flux."""
+    return (f_minus + f_plus - lam*(q_plus - q_minus))/2
+
+
+def num_flux_central(f_minus, f_plus, **kwargs):
+    """Central low-level numerical flux."""
+    return (f_plus + f_minus)/2
+
+
+def hll_flux_driver(state_pair, physical_flux_func,
+                    s_minus, s_plus, normal):
```

Comment:
* Document what `{f,s}_{plus,minus}` are.

Reply:
Ok, see what you think of its current state.  I rearranged it to be consistent with LFR and more encapsulation of concerns.  The docstring was corrected to match the code, and is mostly self-explanatory - but needs more detail with references.